.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -9,10 +9,9 @@ assignees: ''
 
 **Checklist**
 
-- [ ] I verified that the task `./gradlew :dependencyUpdates` works correctly. If that's not the case, it's better to submit a bug-report to https://github.com/ben-manes/gradle-versions-plugin 
-- [ ] I tried to reproduce the bug within `sample-groovy`, `sample-kotlin` or `sample-versionsOnlyMode` 
-- [ ] I have run with `$ ./gradlew --scan buildSrcVersions` or `$./gradlew --scan refreshVersions` to [provide a Build Scan URL](https://dev.to/jmfayard/the-one-gradle-trick-that-supersedes-all-the-others-5bpg)
- 
+- [ ] I tried to reproduce the bug within `sample-kotlin` or another of the samples
```

Comment:
Shouldn't we direct people to try it with the kotlin-libraries-playground instead?

Reply:
Very good point, I update it!